cloudbuild.yaml

```diff
@@ -1,35 +1,14 @@
 steps:
-- name: 'gcr.io/cel-analysis/bazel:ubuntu_20_0_4'
-  entrypoint: bazel
+- name: 'gcr.io/cel-analysis/gcc-9:latest'
   args:
-  - '--output_base=/bazel'
   - 'test'
-  - '--test_output=errors'
-  - '...'
-  id: bazel-test
-- name: 'gcr.io/cel-analysis/bazel:ubuntu_20_0_4'
-  entrypoint: bazel
-  args:
-  - '--output_base=/bazel'
-  - 'test'
-  - '--config=asan'
-  - '--test_output=errors'
   - '...'
-  id: bazel-asan
-- name: 'gcr.io/cel-analysis/bazel:ubuntu_20_0_4'
-  entrypoint: bazel
-  env:
-  - 'CC=gcc'
-  - 'CXX=g++'
-  args:
-  - '--output_base=/bazel'
-  - 'test'
+  - '--compilation_mode=debug'
   - '--test_output=errors'
-  - '...'
-  id: bazel-gcc
+  - '--distdir=/bazel-distdir'
+  - '--show_timestamps'
```

Comment:
I think you still need a clang pipeline. gcc is not that widely used anymore in OSS.